src/Illuminate/Testing/composer.json

```diff
@@ -33,7 +33,9 @@
     "suggest": {
         "illuminate/console": "Required to assert console commands (^7.0).",
         "illuminate/database": "Required to assert databases (^7.0).",
-        "illuminate/http": "Required to assert responses (^7.0)."
+        "illuminate/http": "Required to assert responses (^7.0).",
+        "mockery/mockery": "Required to use mocking (^1.3.1).",
+        "phpunit/phpunit": "Required to use assertions and run tests (^8.4|^9.0)."
```

Comment:
What's the point of this when it's already set under `"require"`

Reply:
WHoops. Meant to delete them.